docs/NRC Endpoints.md

```
@@ -0,0 +1,388 @@
+type VersionFeatures struct {
+    SupportsHealthEndpoint bool  // Can use health endpoint
+    SupportsMultiplexing   bool  // Actions are prefixed with game ID
+    SupportsCustomRouting  bool  // Supports custom routing features
+}
+```
+
+## Integration Flow
+
+### For NeuroRelay-Compatible Integrations
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)